web/src/components/Header.svelte

```diff
@@ -39,11 +53,13 @@
     display: flex;
     justify-content: space-between;
     padding: 8px 16px;
+    gap: 20px;
   }
 
   h1 {
     font-size: 22px;
     margin: 0;
+    margin-right: auto;
```

Comment:
Small css update to align options at the end of their container
screenshot: 
<img width="1792" alt="Screenshot 2023-09-10 at 19 15 38" src="https://github.com/mskelton/ratchet/assets/14045559/1c90e2b1-84e9-4cd3-ad40-ac71fac00d05">